### PR TITLE
[WiP] comparing oc and nc differences

### PR DESCRIPTION
**Pull request by MahdiBaghbani**

### CODE REVIEW

File: lib/Notifier/ScienceMeshNotifier.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ public function getName(): string {
 	 * @throws AlreadyProcessedException When the notification is not needed anymore and should be deleted
 	 */
 
-	public function prepare(INotification $notification, string $languageCode): INotification {
```
Comment (MahdiBaghbani) [inline lib/Notifier/ScienceMeshNotifier.php:72]: is it necessary? @parhamin2010

Comment (MahdiBaghbani) [reply lib/Notifier/ScienceMeshNotifier.php:72]: I mean does Nextcloud really need the `string` type annotation?

File: lib/Controller/SettingsController.php [hunk 2]
```diff
@@ -32,7 +32,7 @@ class SettingsController extends Controller
 	private $serverConfig;
 	private $sciencemeshConfig;
 	private $userId;
```
Comment (MahdiBaghbani) [inline lib/Controller/SettingsController.php:34]: correct for is lowercase `userId`

**Comment by michielbdejong** [comment]
This PR now compares NC for Network Beta (`sciencemesh`) with OC-10 for Network Beta (`oc-10`)

**Comment by MahdiBaghbani** [comment]
@michielbdejong this PR is no longer required, right?

**Pull request closed without merging**